htdocs/install/mysql/tables/llx_societe.sql

```diff
@@ -104,6 +104,8 @@ create table llx_societe
   fk_barcode_type          integer NULL   DEFAULT 0,                    -- barcode type
   price_level              integer NULL,                        		-- level of price for multiprices
   outstanding_limit	       double(24,8)   DEFAULT NULL,					-- allowed outstanding limit
+  id_risk              integer NULL,                        		-- risk level of outstanding
+  max_payment_days              integer NULL,                        		-- maximum number of payment days
```

Comment:
What is the max day of payments if we already have a due date of payment on an invoice ?

Reply:
For the limit of the days, there are good customers but it delivers checks with deadlines, one must have privileges for some by granting him to pay with a deadline of 3 months max and other customers whom they do not deserve to benefit than 15 days because they benefit from a discount ceiling so a reduced margin ...

Pls see above

Reply:
Other risk-level use, the collection manager when he notices that he has a client with a high risk he must put it in his priorities and must find arrangements to avoid the case of unpaid and avoid going through justice .... I hope I was clear to explain the objectives :)

Reply:
The delay to pay an invoice is defined by the payment term (so 15 days, 30 days at end of month, etc...)
If you decide that a given customer can have 45 days to pay its invoices, why not just setting a payment term of 45 days as the default payment term for this customer.

Reply:
@eldy 
the value of max_payment_days should top that of the payment term of each invoice and should never be exceeded!
it's a kind of instruction never to exceed this max_payment_days
if the payment term is not specified in the invoice, it will default to the value of max_payment_days if it's specified.

Reply:
The field "Encours" in prestashop is also already implemented on the customer card of third-party. The field is called "Max. for outstanding bill"

The payment term is also a mandatory information on each invoice, so there is always a max delay already defined on all invoices. And if you want to be sure that customer C has a delay of payment never higher than X, why don't you set a payment term that is limited to X days ?
Currently, i still don't see the difference between the max delay to pay that is currently defined on each invoice and the max_payment_days that you want to introduce ?